tests/python_client/testcases/test_index.py

```diff
@@ -139,6 +139,41 @@ def test_index_params_invalid(self, get_invalid_index_params):
                                    check_task=CheckTasks.err_res,
                                    check_items={ct.err_code: 1, ct.err_msg: ""})
 
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.parametrize("index_name", ["_1ndeX$", "In_$0"])
+    def test_index_naming_rules(self, index_name):
+        """
+        target: test index naming rules
+        method: 1. connect milvus
+                2. Create a collection
+                3. Create an index with an index_name which uses all the supported elements in the naming rules
+        expected: Index create successfully
+        """
+        self._connect()
+        collection_w = self.init_collection_wrap()
+        collection_w.create_index(default_field_name, default_index_params, index_name=index_name)
```

Comment:
Here no check for index name: two ways: 
1. check index name directly after create_index
2. Add check index property check function

Reply:
> Here no check for index name: two ways:
> 
> 1. check index name directly after create_index
> 2. Add check index property check function

The check for index name has been added.